interface/reports/patient_list_creation.php

```diff
@@ -246,582 +319,773 @@ function srch_option_change(elem) {
                 } else {
                     echo "(" . xlt('All') . ")";
                 }
-            }  ?></span>
+            } ?></span>
             <span style="margin-left:5px;"><strong><?php echo xlt('Option'); ?>:</strong>&nbsp;<?php echo text($_POST['srch_option'] ?? '');
             if (!empty($_POST['srch_option']) && ($_POST['srch_option'] == "Insurance Companies") && ($_POST['insurance_companies'] != "")) {
                 if (isset($insarr[$_POST['insurance_companies']])) {
                     echo "(" . text($insarr[$_POST['insurance_companies']]) . ")";
                 } else {
                     echo "(" . xlt('All') . ")";
                 }
-            }  ?></span>
+            } ?></span>
             </p>
         </div>
         <form name='theform' id='theform' method='post' action='patient_list_creation.php' onSubmit="return Form_Validate();">
             <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+            <input type='hidden' name='form_csvexport' id='form_csvexport' value=''/>
             <div id="report_parameters">
                 <input type='hidden' name='form_refresh' id='form_refresh' value=''/>
                 <table>
                     <tr>
-                    <td width='640px'>
-                        <div class="cancel-float" style='float: left'>
-                        <table class='text'>
-                            <tr>
-                                <td class='col-form-label'><?php echo xlt('From'); ?>: </td>
-                                <td><input type='text' class='datetimepicker form-control' name='date_from' id="date_from" size='18' value='<?php echo attr(oeFormatDateTime($sql_date_from, 0, true)); ?>'>
-                                </td>
-                                <td class='col-form-label'><?php echo xlt('To{{range}}'); ?>: </td>
-                                <td><input type='text' class='datetimepicker form-control' name='date_to' id="date_to" size='18' value='<?php echo attr(oeFormatDateTime($sql_date_to, 0, true)); ?>'>
-                                </td>
-                                <td class='col-form-label'><?php echo xlt('Option'); ?>: </td>
-                                <td class='col-form-label'>
-                                    <select class="form-control" name="srch_option" id="srch_option"
-                                        onchange="srch_option_change(this)">
-                                        <?php foreach ($search_options as $skey => $svalue) { ?>
```

Comment:
as discussed above, would not get rid of the translated labels in the $search_options array